### ISSUE TITLE
none: Failed to setup certs: open /usr/share/ca-certificates/minikubeCA.pem: no such file or directory

### ISSUE THREAD

**Issue by MkumaranLakshmi** [kubernetes/minikube#6542]
**The exact command to reproduce the issue**:

[root@terraform ~]# minikube start --vm-driver=none

**The full output of the command that failed**:<details>

[root@terraform ~]# minikube version
minikube version: v1.7.1
commit: 7de0325eedac0fbe3aabacfcc43a63eb5d029fda
[root@terraform ~]# minikube start --vm-driver=none
* minikube v1.7.1 on Centos 7.5.1804
* Using the none driver based on user configuration
* Running on localhost (CPUs=4, Memory=3790MB, Disk=28862MB) ...
* OS release is CentOS Linux 7 (Core)
! Node may be unable to resolve external DNS records
* Preparing Kubernetes v1.17.2 on Docker '19.03.5' ...
* Downloading kubeadm v1.17.2
* Downloading kubelet v1.17.2
* Downloading kubectl v1.17.2
*
X Failed to setup certs: Copy /root/.minikube/ca.crt: error creating file at /usr/share/ca-certificates/minikubeCA.pem: open /usr/share/ca-certificates/minikubeCA.pem: no such file or directory
*
* minikube is exiting due to an error. If the above message is not useful, open an issue:
  - https://github.com/kubernetes/minikube/issues/new/choose
[root@terraform ~]# ll -al /root/.minikube/ca.crt

</details>

**The output of the `minikube logs` command**:<details>

</details>

**The operating system version**:

**Comment by tstromberg** [comment]
Thanks for the bug report!

 I wonder if broke usage of the none driver on nodes where it had been used previously. Do you mind including the output of:

`minikube start --vm-driver=none --alsologtostderr`

Thanks!

**Comment by tstromberg** [comment]
Any chance that `/usr/share/ca-certificates` does not exist on your system?

I ask because we appear to make that assumption in v1.7:

```
I0207 07:47:32.468438   70351 ssh_runner.go:155] Checked if /usr/share/ca-certificates/minikubeCA.pem exists, but got error: Process exited with status 1
I0207 07:47:32.468707   70351 ssh_runner.go:174] Transferring 1066 bytes to /usr/share/ca-certificates/minikubeCA.pem
I0207 07:47:32.469290   70351 ssh_runner.go:193] minikubeCA.pem: copied 1066 bytes
```

It seems easy enough to fix if so.

**Comment by arjantop** [comment]
The of none driver is broken even if starting from scratch.
We use Amazon Linux, there is no `/usr/share/ca-certificates`.

**Comment by zigarn** [comment]
Same on CentOS 7 AMI: `/usr/share/ca-certificates` does not exists.
Creating it by hand solve this problem.

**Comment by zigarn** [comment]
I confirm that the problem is solved in `v1.7.2`
